cmd/juju/application/deploy_test.go

```diff
@@ -495,6 +495,12 @@ func (s *DeploySuite) TestForceMachine(c *gc.C) {
 	s.assertForceMachine(c, machine.Id())
 }
 
+func (s *DeploySuite) TestInvalidSeriesForModel(c *gc.C) {
+	ch := testcharms.Repo.CharmArchivePath(s.CharmsPath, "dummy")
+	err := runDeploy(c, ch, "portlandia", "--series", "kubernetes")
+	c.Assert(err, gc.ErrorMatches, `series "kubernetes" in a non CAAS model not valid`)
```

Comment:
Don't say CAAS. I'd say "kubernetes"

Reply:
yeah, that occurred to me also. i thing "container" is better though